guides/v2.2/magento-functional-testing-framework/release-2/test/annotations.md

```diff
@@ -195,6 +196,26 @@ Attribute|Type|Use
 <useCaseId value="USECASE-1"/>
 ```
 
+### skip
+
+The `<skip>` element is an implementation that allows a test to be skipped. It also specifies the issue(s) that cause the test to be skipped in `<issueId value="issue#">` elements.
```

Comment:
_The `<skip>` element is used to skip a test.
It contains one or more child elements `<issueId>` to specify one or more issues that cause the test skipping._